.github/workflows/daily-test.yml

```diff
@@ -47,6 +50,17 @@ jobs:
             sleep 10
           done
 
+      - name: Run test against the latest 2 iOS versions using no-video-codecs build
+        id: tests-no-video-codecs
+        run: |
+          id=$(curl -F 'payload=@amazon-chime-sdk-app-no-video-codecs.ipa' -F name=amazon-chime-sdk-app-no-video-codecs.ipa -u "${{ secrets.SAUCE_USERNAME }}:${{ secrets.SAUCE_ACCESS_KEY }}" 'https://api.us-west-1.saucelabs.com/v1/storage/upload' |jq '.item.id')
+          npm install
+          npm run build
+          for os in 15 16; do
```

Comment:
Should be OS 17 and 18

Reply:
need appium upgrade: https://taskei.amazon.dev/tasks/V1564443134